backend/api/admin_views.py

```
@@ -0,0 +1,204 @@
+            except Exception as e:
+       
+                target_user.role = old_role
+                target_user.save()
+                return Response({"error": f"Failed to update Firebase: {str(e)}"}, 
+                              status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+                
+        except Exception as e:
+            return Response({"error": "Invalid credentials"}, status=status.HTTP_400_BAD_REQUEST)
+
+class VideoManagementView(APIView):
+    permission_classes = [IsAuthenticated, IsAdmin]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)